harbour-nayttamo.pro

```diff
@@ -17,14 +17,29 @@ CONFIG += sailfishapp libcrypto
 SOURCES += src/harbour-nayttamo.cpp \
     src/urldecrypt.cpp
 
-OTHER_FILES += \
-    qml/cover/CoverPage.qml \
-    qml/pages/FirstPage.qml \
+DISTFILES += \
```

Comment:
This seems nice but what's the reasoning with `DISTFILES` vs `OTHER_FILES` ?

Reply:
Those both work but having files split into two variables is confusing and I selected the DISTFILES based on what was being used on some other apps.